tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml

```diff
@@ -1,17 +1,176 @@
+                    id="container_LANI4JGBE"
+                    marginEnabled="false"
+                    paddingEnabled="false"
+                    persistColorWhenThemeSwitches="false"
+                    title="Barchart Showcase">
+                    <title_1
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="typerefinery/components/content/title"
+                        borderEnabled="false"
+                        classes="\0"
+                        headingLevel="h6"
+                        headingSize="h5"
+                        id="title_1_UFZHBDTYV"
+                        marginBottom="four"
+                        marginEnabled="true"
+                        marginTop="five"
+                        paddingEnabled="false"
+                        persistColorWhenThemeSwitches="false"
+                        textAlignment="center"
+                        title="Line chart  using data source"/>
+                    <chart
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="typerefinery/components/widgets/chart"
+                        borderEnabled="false"
+                        classes="\0"
+                        dataSource="/apps/typerefinery/components/widgets/chart/clientlibs/variants/barchart/mock/datasource1.json"
+                        flowapi_createdon="\0"
+                        flowapi_designtemplate="\0"
+                        flowapi_editurl="\0"
+                        flowapi_enable="false"
+                        flowapi_flowstreamid="\0"
+                        flowapi_group="\0"
+                        flowapi_httproute="\0"
+                        flowapi_iscontainer="\0"
+                        flowapi_sampledata="/apps/typerefinery/components/widgets/chart/templates/flowsample.json"
+                        flowapi_template="/apps/typerefinery/components/widgets/chart/templates/chart.json"
+                        flowapi_title="\0"
+                        flowapi_topic="\0"
+                        flowapi_updatedon="\0"
+                        id="chart_HAG2AO4YM"
+                        marginEnabled="false"
+                        paddingEnabled="false"
+                        persistColorWhenThemeSwitches="false"
+                        websocketHost="\0"
+                        websocketTopic="\0"/>
+                </container_2>
+                <container_2_1
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="typerefinery/components/layout/container"
+                    borderEnabled="false"
+                    classes="\0"
+                    flexEnabled="false"
+                    id="container_LANI4JGBE"
+                    marginEnabled="false"
+                    paddingEnabled="false"
+                    persistColorWhenThemeSwitches="false"
+                    title="Barchart Showcase">
```

Comment:
This is for Line chart why is this Barchart here ?

Reply:
also title should just be Barchart, there is no reason to append " Showcase" as its given.

Reply:
Done, with single container having section with title